src/orquestra/sdk/_base/_workflow.py

```diff
@@ -158,42 +158,50 @@ def local_run(self) -> _R:
         _dsl.DIRECT_EXECUTION = False
         return result
 
-    def prepare(
+    def run(
         self,
-        config: Union[_api.RuntimeConfig, str],
+        config: Optional[Union[_api.RuntimeConfig, str]] = None,
         project_dir: Optional[Union[str, Path]] = None,
         workspace_id: Optional[WorkspaceId] = None,
         project_id: Optional[ProjectId] = None,
     ) -> _api.WorkflowRun:
         """
-        "Prepares" workflow for running. Call ".start()" on the result to
-        schedule the workflow for execution.
+        Schedules workflow for execution.
 
         Args:
-            config: SDK needs to know where to execute the workflow. The config
-                contains the required details. This can be a RuntimeConfig object, or
-                the name of a saved configuration.
+            config: SDK needs to know where to execute the workflow. This
+                objects contains the required details.
             project_dir: the path to the project directory. If omitted, the current
                 working directory is used.
             workspace_id: ID of the workspace for workflow - supported only on CE
             project_id: ID of the project for workflow - supported only on CE
 
         Raises:
-            ConfigNameNotFoundError: when the configuration has not been saved prior to
-                this point.
             orquestra.sdk.exceptions.DirtyGitRepo: (warning) when a task def used by
                 this workflow def has a "GitImport" and the git repo that contains it
                 has uncommitted changes.
-            ProjectInvalidError: when only 1 out of project and workspace is passed
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏What happens if I provide only a project?

Reply:
Same as above - github shows it weirdly, although you are right we can get this exception in here, so Im adding the docs